modules/core/shared/src/main/scala/lucuma/core/enum/GmosDetector.scala

```diff
@@ -16,18 +23,35 @@ sealed abstract class GmosDetector(
   val tag: String,
   val shortName: String,
   val longName: String,
-  val northPixelSize: Angle,
-  val southPixelSize: Angle,
-  val shuffleOffset: Int,
-  val xSize: Int,
-  val ySize: Int,
-  val maxRois: Int
-) extends Product with Serializable
+  val shuffleOffset: Quantity[PosInt, Pixels],
+  val maxRois: PosInt
+) extends Product with Serializable {
+  def xSize(site: Site): Quantity[PosInt, Pixels]
+  def ySize(site: Site): Quantity[PosInt, Pixels]
+  def pixelSize(site: Site): Angle
```

Comment:
Would it make sense to group these 3 values into another structure? (Maybe not, I don't know the model that well)

Reply:
Maybe it would be simpler to have three instances `G[N|S]Hammamtsu` @swalker2m WDYT?